lib/provider.js

```diff
@@ -253,7 +252,11 @@ const parseOutput = (text) => {
   text = text && text.slice(0, text.indexOf('\n'));
 
   if (!text) {
-    throw noOutputFromOracleError();
+    atom.notifications.addWarning('Elm Autocomplete Error', {
```

Comment:
Ok, but then we would need to catch the case that the editor has a filePath but is not saved yet (as happens when creating editors with [advanced-open-file](https://atom.io/packages/advanced-open-file)). That's actually where this PR came from. This error was thrown for every letter that I inserted :smile: .